src/main/java/dk/kb/ds/cumulus/export/CalendarUtils.java

```diff
@@ -26,16 +20,14 @@
     /** constructor to prevent instantiation of utility class. */
     protected CalendarUtils() {}
 
-    /** A single instance of the DatatypeFactory to prevent overlap from recreating it too often.*/
-    private static DatatypeFactory factory = null;
-
     private static final String DATE_RANGE_PATTERN = "[%s TO %s]";
     private static final String DASH = "-";
 
     /**
      * ISO8601 representation with second granularity and Zulu time. Compatible with Solr datetime.
      */
-    private static final DateTimeFormatter ISO8601_FORMATTER = DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss'Z'");
+    private static final DateTimeFormatter ISO8601_FORMATTER =  DateTimeFormatter.
```

Comment:
As we are not using the Locale (the formatting string does not contain locale-dependent parameters), setting the Locale for the `DateTimeFormatter.ofPattern` is "only" to signal to forbiddenapis that we know what we are doing. [Locale.ROOT](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/Locale.html#ROOT) is used for such purposes.